services/autoscaling/src/simcore_service_autoscaling/utils/auto_scaling_core.py

```diff
@@ -148,3 +148,34 @@ def find_selected_instance_type_for_task(
         raise Ec2InstanceInvalidError(msg=msg)
 
     return selected_instance
+
+
+def get_machine_buffer_type(
+    available_ec2_types: list[EC2InstanceType],
+) -> EC2InstanceType:
+    return available_ec2_types[0]
+
+
+DRAINED_NODES = list[AssociatedInstance]
```

Comment:
`TypeAlias`
Why type alias is written as a constant here?

Reply:
yes, added assert